prov/gni/include/gnix.h

```diff
@@ -379,7 +380,8 @@ struct gnix_fab_req_msg {
 struct gnix_fab_req {
 	struct slist_entry        slist;
 	uint64_t tag;
-	uint64_t mask_bits;
+	uint64_t ignore_bits;
+	uint64_t addr_ignore_bits;
```

Comment:
If this isn't applicable, then it will be removed. I was under the assumption that there was a case where you might want to receive a message from a subset of the endpoints and not all of them. UNSPEC covers the case where it would match any EP, however, is it possible that the provider would want to match against a subset of the EPs? If the answer to this is no, then it's a simple removal of an existing feature in the tag matching implementation.